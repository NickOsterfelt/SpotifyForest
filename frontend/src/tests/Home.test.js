import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import Home from "../routes/Home";


it("renders without crashing", function () {
    render(
        <MemoryRouter>
                <Home />
       </MemoryRouter>
    );
});

it("matches snapshot", function () {
    const { asFragment } = render(
        <MemoryRouter>
                <Home />
        </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
});
