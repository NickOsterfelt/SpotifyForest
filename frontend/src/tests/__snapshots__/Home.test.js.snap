// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container-fluid"
  >
    <div
      class="row justify-content-center "
    >
      <div
        class="col-sm-4 col-md-3 col-lg-2 px-2 profile-sidebar"
      >
        <h4
          class="text-light mt-3"
        >
          Your profile:
        </h4>
        <div>
          Loading user details...
        </div>
      </div>
      <div
        class="col-sm-4 col-md-6 col-lg-8 middle-section"
      >
        <div>
          <div>
            Loading Group data....
          </div>
        </div>
      </div>
      <div
        class="col-sm-4 col-md-3 col-lg-2 px-2 profile-sidebar"
      >
        <div>
          <div
            class="text-light"
          >
            Click on a user's tile to see more info!
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
